topology/pynest/hl_api.py

```diff
@@ -2115,3 +2115,34 @@ def PlotKernel(ax, src_nrn, mask, kern=None, mask_color='red',
                              'version of PyTopology')
 
     plt.draw()
+
+
+def SelectNodesByMask(layer, xpos, ypos, mask_obj):
+    """
+    Obtain GIDs inside a specified area.
+
+    Parameters
+    ----------
+    layer : tuple/list of int
+        List containing the single layer to get GIDs from.
+    xpos : double
+        Double value specifying x-coordinate of where we start to search. Can
```

Comment:
It makes more sense to pass this position as a tuple or numpy array; in that way, 2D and 3D are covered by the same code. `anchor` could be a good variable name, as it is the position where the mask is anchored.